wire-ios/Wire-iOS/Sources/UserInterface/Helpers/AccentColor+Name.swift

```diff
@@ -16,19 +16,17 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Foundation
-import WireCommonComponents
+import WireDataModel
 
 extension AccentColor {
+
     var name: String {
         typealias AccentColor = L10n.Localizable.Self.Settings.AccountPictureGroup.AccentColor
         switch self {
         case .blue:
             return AccentColor.blue
         case .green:
             return AccentColor.green
-        case .deprecatedYellow:
-            return AccentColor.yellow
```

Comment:
suggestion: I think we should also remove the localisable string

Reply:
https://github.com/wireapp/wire-ios/pull/1332/commits/1b48e0e8080709035043388307cf85db2fd12f2a